lib/manager/cocoapods/artifacts.ts

```diff
@@ -78,12 +79,37 @@ export async function updateArtifacts({
   }
   logger.debug(`Returning updated lockfile: ${lockFileName}`);
   const lockFileContent = await readLocalFile(lockFileName);
-  return [
+  const res = [
     {
       file: {
         name: lockFileName,
         contents: lockFileContent,
       },
     },
   ];
+
+  const podsDir = join(dirname(packageFileName), 'Pods/');
```

Comment:
This isn't crossplatform. I know it's unlikely that someone will be running Renovate on Windows, to update dependencies on an iOS project, but better safe than sorry. 

```suggestion
  const podsDir = join(dirname(packageFileName), 'Pods');
```

Reply:
I guess the gomod code should have the same fix as well

Reply:
Yeah 🤦. Don't worry, I'll do that in another PR.